support/agent/cases_test.go

```diff
@@ -62,6 +62,86 @@ func TestCreate20Miners(t *testing.T) {
 	}
 }
 
+func Test500Epochs(t *testing.T) {
```

Comment:
Please document the intention here of a sanity check that is fast enough to run during dev cycles and CI.